onto_tool/ontograph.py

```diff
@@ -404,6 +404,8 @@ def gather_instance_info(self):
             predicate_usage = list(self.select_query(query_text))
             logging.debug("%s items returned for %s", len(predicate_usage), predicate)
             for usage in predicate_usage:
+                if 'tgt' in usage and usage['tgt'] is None:
+                    del usage['tgt']
                 if 'src' not in usage or usage['src'] is None or int(usage.get('num', 0)) < self.threshold:
                     continue
                 self.record_predicate_usage(predicate, predicate_str, usage)
```

Comment:
In the method `self.record_predicate_usage`, the following variable gets set:
```
tgt_uri = self.deepest_class(usage['tgt']) if 'tgt' in usage else None
```

Sometimes `usage['tgt']` will be equal to `None`. The script then fails in `self.deepest_class` when trying to split the expected string in `usage['tgt']`.